client/js/widgets/widget.js

```diff
@@ -603,7 +603,9 @@ export class Widget extends StateManaged {
     for(const original of routine) {
       let a = JSON.parse(JSON.stringify(original));
       if(typeof a == 'object')
-        a = evaluateVariablesRecursively(a);
+        a = evaluateVariablesRecursively(a)
+      else
+        a = original.trim();
```

Comment:
This does mean that leading spaces before a variable expression are now allowed where they previously were not. (I don't care, but someone might.)

Reply:
Yes. I meant to mention this in my comment but forgot. I actually think it's a feature, not a bug.